docs/DevelopmentGuide.md

```diff
@@ -149,6 +121,88 @@ bundle exec rake dependency:lock['/app/gemfiles/ruby_3.3_stripe_*.gemfile']
 bundle exec rake dependency:lock['/app/gemfiles/ruby_3.3_stripe_latest.gemfile']
 ```
 
+**How to add new dependency group**
```

Comment:
I wonder if we should also leave a warning to avoid creating new groups when possible... In particular, we're exploding the number of files in the repo + need to be updated + that show up on a diff (making github not able to render the diff anymore in some cases).

So something like "Consider reusing an existing group if possible, as maintaining too many groups causes trouble in CI/git"?

Reply:
Done